Source/AssetRipper.Processing/AnimationClips/AnimationClipConverter.cs

```diff
@@ -207,53 +217,19 @@ private void ProcessConstant(IClip clip, IConstantClip constant, IAnimationClipB
 					}
 					else if (binding.CustomType == (byte)BindingCustomType.None)
 					{
-						AddDefaultCurve(binding, path, time, constant.Data[curveIndex]);
+						AddDefaultCurve(binding, path, time, constant.Data[curveIndex], 0, 0);
 						curveIndex++;
 					}
 					else
 					{
-						AddCustomCurve(bindings, binding, path, time, constant.Data[curveIndex]);
+						AddCustomCurve(binding, path, time, constant.Data[curveIndex], 0, 0);
```

Comment:
I think I would have preferred in and out slope to have default values in method definitions.

Reply:
implemented